committee-steering/governance/sig-governance.md

```diff
@@ -1,16 +1,24 @@
 # SIG Roles and Organizational Governance
 
-This charter adheres to the conventions described in the [Kubernetes Charter README].
+This charter adheres to the conventions described in the [Kubernetes Charter README]. It will be updated as needed to meet the current needs of the Kubernetes project.
 
-This document will be updated as needed to meet the current needs of the Kubernetes project.
+In order to standardize Special Interest Group efforts, create maximum transparency, and route contributors to the appropriate SIG, SIGs should follow these guidelines:
+
+- Create a charter and have it approved according to the [SIG charter process]
+- Meet regularly, at least for 30 minutes every 3 weeks, except November and December
+- Keep up-to-date meeting notes, linked from the SIG's page in the community repo
+- Record meetings and make them publicly available on a YouTube playlist
+- Report activity in the weekly community meeting at least once every quarter
+- Participate in release planning meetings and retrospectives, and burndown meetings, as needed
+- Ensure related work happens in a project-owned github org and repository, with code and tests explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response, bug fixes, etc.
+- Use the [forums provided] as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings
```

Comment:
Should we add something about having a clearly defined group of deciders for sub projects?

Reply:
wouldn't hurt